lib/kanban/issues.ex

```diff
@@ -0,0 +1,48 @@
+defmodule Kanban.Issues do
+  @moduledoc """
+  The process of Issues.
+  """
+
+  import Ecto.Changeset
+
+  alias Kanban.Data.{Repo, Issue}
+
+  use GenServer, restart: :transient
+
+  def start_link() do
+    GenServer.start_link(__MODULE__, nil, name: __MODULE__)
+  end
+
+  @impl GenServer
+  def init(state), do: {:ok, state}
+
+  @impl GenServer
+  def handle_call({:get}, _from, state) do
```

Comment:
Tuple тут, наверное, не очень нужен.
```suggestion
  def handle_call(:get, _from, state) do
```